### DISCUSSION TITLE
Using external redis for ArgoCD

### DISCUSSION THREAD

**Discussion by ApoorwaAtre** [argoproj/argo-cd#15336] [Q&A]
I am trying to replace ArgoCD Redis with Amazon MemoryDB  Redis. When I deploy the Argo CD HELM chart, I disable argocd Redis & update the host, but unable to see external redis pod after installing Argocd

**Comment by FernandoMiguel** [comment]
why would you expect a pod if you disabled redis?
the remaining pods will connect directly to AWS redis, instead of the pod previously deployed by argo
